### PR TITLE
chore(RELEASE-1269): default to RHBA for advisory

### PR DESCRIPTION
**Pull request by scoheb**
- new default of RHBA for advisories when a type is not provided.

### CODE REVIEW

File: tasks/create-advisory/create-advisory.yaml [hunk 1]
```diff
@@ -65,7 +66,15 @@ spec:
         configMapName=$(jq -er '.sign.configMapName' "$(workspaces.data.path)/$(params.dataPath)")
 
         # Validate type
-        advisoryType=$(jq -r ".type" <<< "$advisoryData")
+        advisoryType=$(jq -r '.type' <<< "$advisoryData")
+        if [ "$advisoryType" == "null" ]; then
```
Comment (johnbieren) [inline tasks/create-advisory/create-advisory.yaml:70]: Any reason why not to just use the jq default mechanism?

Comment (scoheb) [reply tasks/create-advisory/create-advisory.yaml:70]: using the jq default mechanism will only give me `advisoryType` with the default value.

I then need to patch `advisoryData` with the updated value later.

is there a better way?

Comment (johnbieren) [reply tasks/create-advisory/create-advisory.yaml:70]: Great point

**Review by johnbieren — APPROVED** [review]

**Pull request merged**